blobmodel/geometry.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         else:
             self.y = np.arange(0, self.Ly, self.Ly / self.Ny)
         self.t: NDArray[Any, Float[64]] = np.arange(0, self.T, self.dt)
+        self.xx, self.yy, self.tt = np.meshgrid(self.x, self.y, self.t)
```

Comment:
Since now xx, yy, tt are public variables, they should have understandable names

Reply:
done